developers.libra.org/scripts/build_docs.sh

```diff
@@ -43,11 +53,15 @@ while getopts 'hbr' flag; do
       BUILD_RUSTDOCS=true
       ;;
     *)
-      usage
+      usage;
+      exit 0;
       ;;
   esac
 done
 
+# Install Rust (Netlify will need this for website previews, for example)
+install_rustup
```

Comment:
I hope this is enough and we don't have to call the dev-setup.sh script to install all the deps we need to do a full build of libra

Reply:
Yeah. I wanted to start light and see what else we may need to add, if anything.